tools/issue-labeler/src/SearchIndexCreator/Program.cs

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Extensions.Configuration;
+using Octokit;
+using Azure.Identity;
+using Azure.AI.OpenAI;
+using Azure.Search.Documents.Indexes;
+using AzureRAGService;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace SearchIndexCreator
+{
+    public class SearchIndexCreator
+    {
+        public static async Task Main(string[] args)
```

Comment:
We should look to refactor this and use `System.CommandLine` or similar to cut down boilerplate and make maintenance easier.